src/clj/game/cards/resources.clj

```diff
@@ -108,14 +108,13 @@
                             (effect-completed state :runner eid)))}]})
 
 (defcard "Adjusted Chronotype"
-  {:events [{:event :runner-lose
-             :req (req (and (some #{:click} target)
-                            (let [click-losses (count (filter #(= :click %) (mapcat first (turn-events state side :runner-lose))))]
-                              (or (= 1 click-losses)
-                                  (and (= 2 click-losses)
-                                       (has-flag? state side :persistent :genetics-trigger-twice))))))
+  {:events [{:event :runner-click-lose
+             :req (req (let [click-losses (count (filter #(= :click %) (mapcat first (turn-events state side :runner-lose))))]
```

Comment:
Should this be checking `:runner-click-lose`?

Reply:
Yeah that's adjusted chronotype. Leaving it on runner-loss is what causes the bug where it triggers from riot suppression.

At the moment it's moot and I'd prefer to wait on the bioroid ruling, but at some point I'd like to cleanup the way clicks are spent. At the moment, most engine click spending uses (deduct that fires no events at all, presumably to avoid triggering chronotype. Obviously that's not needed any more and you can just use (lose-clicks to trigger abilities and (lose to avoid it, the same as credits or tags.